### PR TITLE
Shorten the long Configuration name

### PR DESCRIPTION
**Pull request by Shee43**
Fixes : #97 
@SarikaSinha

### CODE REVIEW

File: org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/ClasspathShortener.java [hunk 1]
```diff
@@ -294,8 +294,10 @@ private void shortenPathUsingClasspathArgumentFile(int modulePathArgumentIndex,
 		File file;
 		try {
 			String timeStamp = getLaunchTimeStamp();
-			File argFile = new File(processTempFilesDir, String.format(LAUNCH_TEMP_FILE_PREFIX + "%s" + option //$NON-NLS-1$
-					+ "-arg-%s.txt", getLaunchConfigurationName(), timeStamp)); //$NON-NLS-1$
+			String configName = (getLaunchConfigurationName().length() > 10) ? getLaunchConfigurationName().substring(0, 10)
```
Comment (iloveeclipse) [inline org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/ClasspathShortener.java:297]: 1) Please extract getLaunchConfigurationName() to a variable
2) Please extract 10 to a constant

**Review by iloveeclipse — CHANGES_REQUESTED** [review]
Thanks for contributing.

Please look at the commit messages in git history of this repository and provide a proper commit message.

A good commit message should have short first sentence mentioning original issue on the first line and after a new lime longer explanation of the problem/solution. 

Also please add a "Fixes https://github.com/eclipse-jdt/eclipse.jdt.debug/issues/97" line at the end of commit message.

**Comment by iloveeclipse** [comment]
Please also bump org.eclipse.jdt.launching bundle version to 3.19.1000 in manifest and pom.xml.

**Comment by SarikaSinha** [comment]
In the favour of https://github.com/eclipse-jdt/eclipse.jdt.debug/pull/232

**Pull request closed without merging**